docs-web/installation/installation-with-manifests.md

```diff
@@ -62,18 +62,20 @@ In this section, we create resources common for most of the Ingress Controller i
 ValidationError(CustomResourceDefinition.spec): unknown field "preserveUnknownFields" in io.k8s.apiextensions-apiserver.pkg.apis.api extensions.v1beta1.CustomResourceDefinitionSpec
 ```
 
+**Note**: There are two different sets of Custom Resources, one for Kubernetes <= v1.15 and one for Kubernetes >= v1.16. For Kubernetes <= v1.15 substitute `crds` with `crds-v1beta1` in the following commands.
```

Comment:
do we really need to use crds v1 for Kubernetes >= 1.15? I wonder if can just use v1beta until we stop supporting k8s 1.15. This way we will not need to use different folders in the docs and make any comments 🤔 

we can still store v1 in some folder if those are necessary